### PR TITLE
Some general code style improvements

### PR DESCRIPTION
**Pull request by paulinevos**
Adding camel case and removing a bunch of unused variables. This is just the result of working on an actual issue, but that branch was becoming quite big so sumbitting this separately (also it's october lol)

### CODE REVIEW

File: src/Controller/UsersController.php [hunk 1]
```diff
@@ -466,25 +464,6 @@ public function setTrusted(Request $request, PDO $db)
         $view->setResponseCode(Http::NO_CONTENT);
     }
 
-    /**
```
Comment (paulinevos) [inline src/Controller/UsersController.php]: Not sure why these changes are in here, will clean (will be part of other MR)

File: .gitignore [hunk 2]
```diff
@@ -11,3 +11,7 @@ vendor/.git
 node_modules
 vendor
 diff.txt
+.phpcs-cache
+.phpunit.result.cache
+tests/.phpunit.results.cache
```
Comment (localheinz) [inline .gitignore]: This should not be necessary, see:

https://github.com/joindin/joindin-api/blob/f7dede99cbe0c325565d64626999c4797ef4d68f/phpunit.xml.dist#L11:

File: .gitignore [hunk 3]
```diff
@@ -11,3 +11,7 @@ vendor/.git
 node_modules
 vendor
 diff.txt
+.phpcs-cache
```
Comment (localheinz) [inline .gitignore]: This should not be necessary, see:

https://github.com/joindin/joindin-api/blob/f7dede99cbe0c325565d64626999c4797ef4d68f/composer.json#L67

File: .gitignore (hunk 2 above)
Comment (paulinevos) [reply .gitignore]: Ah, I think I still had an unstaged cache file from months back before that was implemented :)

Will drop the commit.
